_data/toc.yml

```diff
@@ -10,6 +10,7 @@
 - Marketing
 - Summit
 - Support:
+  - Partner Guide
```

Comment:
When you add a new entry to the toc, it must match the name of the file, for example `Partner Guide` looks for the folder `partner-guide` so you can either change the Parnter Guide to Partner on the TOC or you can change the folder name